src/main/java/org/fcrepo/spec/testsuite/versioning/LdprvHttpGet.java

```diff
@@ -62,14 +62,27 @@ public void shouldReturn406WhenNoLdprm() {
         confirmPresenceOfTimeGateLink(response);
 
         //get timegate uri
+        final URI timemapUri = getTimeMapUri(response);
+        //check for any mementos on the time gate (they will exist if repository auto versions objects by default.
+        final Response timeMapResponse = doGet(timemapUri.toString(),
+                new Header("Accept", "application/link-format"));
+        final boolean hasMemento;
+        if (timeMapResponse.getBody().asString().contains("rel=\"memento\"")) {
+            hasMemento = true;
+        } else {
+            hasMemento = false;
+        }
+
         final URI timeGateUri = getTimeGateUri(response);
 
         //query timegate using Accept-Datetime
         final String isoDateString = "1970-01-01T00:00:00Z";
 
         final String rfc1123Date = convertToRfc1123DateTimeString(isoDateString);
 
-        doGetUnverified(timeGateUri.toString(), new Header("Accept-Datetime", rfc1123Date)).then().statusCode(406);
+        //if no mementos created verify 406 otherwise 302
+        doGetUnverified(timeGateUri.toString(), new Header("Accept-Datetime", rfc1123Date)).then()
```

Comment:
I verified that this also passes when autoversioning is set to false, which is great.